From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Fri, 17 Apr 2026 18:00:05 +0800
Subject: Use 4k blocksize in all ext4 mke2fs.conf such that lvm migration

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1817097

 between non-4k PVs and 4k PVs works irrespective of the volume size.
---
 misc/mke2fs.conf.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index b7fc95d..e90802d 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -14,11 +14,9 @@
 		features = has_journal,extent,huge_file,flex_bg,metadata_csum,metadata_csum_seed,64bit,dir_nlink,extra_isize,orphan_file
 	}
 	small = {
-		blocksize = 1024
 		inode_ratio = 4096
 	}
 	floppy = {
-		blocksize = 1024
 		inode_ratio = 8192
 	}
 	big = {
